.github/workflows/postman-collection-runner.yml

```diff
@@ -1,68 +1,124 @@
-name: Postman Collection Runner
+name: Run collections against your PRs
 
 on:
   workflow_dispatch:
-  schedule:
-    - cron: "30 0,12 * * *" # Run workflow at 6 AM and 6 PM IST
+  merge_group:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  CARGO_INCREMENTAL: 1
+  CARGO_NET_RETRY: 10
+  RUSTUP_MAX_RETRIES: 10
+  RUST_BACKTRACE: short
+  CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
+
 jobs:
-  api_tests:
-    name: Run Postman Collection on integ env
+  Setup:
+    name: Setup Rust and Server
     runs-on: ubuntu-latest
 
     strategy:
       fail-fast: false
       matrix:
         connector:
-          - aci
-          # - authorizedotnet
-          - bluesnap
-          # - braintree
-          - checkout
-          # - iatapay
-          # - multisafepay
-          - nmi
-          - stripe
-          # - trustpay
-          - worldline
+          - aci, checkout, stripe
```

Comment:
verify that this is what you want? syntax may be wrong here.

Reply:
After discussion, intention is to pass multiple `,` separated values to each job to reduce the startup/runtime.

Reply:
Yea, this reduces startup time and hence allows us to reuse already set-up server for upcoming connectors.   
With this, we could effectively reduce the run time from a mere 16+ minutes to ~7 minutes